standard/classes.md

```diff
@@ -1583,7 +1583,7 @@ Constants are permitted to depend on other constants within the same program as
 > }
 > ```
 >
-> the compiler first evaluates `A.Y`, then evaluates `B.Z`, and finally evaluates `A.X`, producing the values `10`, `11`, and `12`.
+> a compiler first evaluates `A.Y`, then evaluates `B.Z`, and finally evaluates `A.X`, producing the values `10`, `11`, and `12`.
```

Comment:
Again, perhaps this should be "must first evaluate"? (It sounds odd with "a compiler first evaluates" to me.)